### ISSUE TITLE
[BUG] Cannot consume document when drag and drop to web interface

### ISSUE THREAD

**Issue by Lunaden** [paperless-ngx/paperless-ngx#1682]
### Description

When trying to drag and drop file to webui sometimes file is consume with success and sometimes not (same file).
![image](https://user-images.githubusercontent.com/34348877/192790529-75ea60fd-658d-4e46-82e0-a1a72ef5987c.png)

![image](https://user-images.githubusercontent.com/34348877/192790477-ee67eb3d-72b1-46cf-8186-9e21c7982d85.png)

```
Invoice-572B7446-0006.pdf: Cannot consume /tmp/paperless/paperless-upload-5jc7vf_c: File not found. : Traceback (most recent call last):
  File "/usr/src/paperless/src/src/django-q/django_q/cluster.py", line 454, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 154, in consume_file
    document = Consumer().try_consume_file(
  File "/usr/src/paperless/src/documents/consumer.py", line 264, in try_consume_file
    self.pre_check_file_exists()
  File "/usr/src/paperless/src/documents/consumer.py", line 106, in pre_check_file_exists
    self._fail(
  File "/usr/src/paperless/src/documents/consumer.py", line 90, in _fail
    raise ConsumerError(f"{self.filename}: {log_message or message}") from exception
documents.consumer.ConsumerError: Invoice-572B7446-0006.pdf: Cannot consume /tmp/paperless/paperless-upload-5jc7vf_c: File not found.
```

### Steps to reproduce

Open the Web Ui
Drag and drop a file or upload document from file explorer.

### Webserver logs

```bash
15:04:12 [Q] INFO Process-1:84 processing [Invoice-572B7446-0006.pdf]
[2022-09-28 15:04:13,118] [ERROR] [paperless.consumer] Cannot consume /tmp/paperless/paperless-upload-3kuv4ir5: File not found.
15:04:13 [Q] INFO Process-1:84 stopped doing work
15:04:13 [Q] ERROR Failed [Invoice-572B7446-0006.pdf] - Invoice-572B7446-0006.pdf: Cannot consume /tmp/paperless/paperless-upload-3kuv4ir5: File not found. : Traceback (most recent call last):
  File "/usr/src/paperless/src/src/django-q/django_q/cluster.py", line 454, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 154, in consume_file
    document = Consumer().try_consume_file(
  File "/usr/src/paperless/src/documents/consumer.py", line 264, in try_consume_file
    self.pre_check_file_exists()
  File "/usr/src/paperless/src/documents/consumer.py", line 106, in pre_check_file_exists
    self._fail(
  File "/usr/src/paperless/src/documents/consumer.py", line 90, in _fail
    raise ConsumerError(f"{self.filename}: {log_message or message}") from exception
documents.consumer.ConsumerError: Invoice-572B7446-0006.pdf: Cannot consume /tmp/paperless/paperless-upload-3kuv4ir5: File not found.
15:04:16 [Q] INFO recycled worker Process-1:84
15:04:16 [Q] INFO Process-1:88 ready for work at 544
15:06:06 [Q] INFO Process-1:85 processing [Invoice-572B7446-0006.pdf]
[2022-09-28 15:06:07,174] [ERROR] [paperless.consumer] Cannot consume /tmp/paperless/paperless-upload-5jc7vf_c: File not found.
15:06:07 [Q] INFO Process-1:85 stopped doing work
15:06:07 [Q] ERROR Failed [Invoice-572B7446-0006.pdf] - Invoice-572B7446-0006.pdf: Cannot consume /tmp/paperless/paperless-upload-5jc7vf_c: File not found. : Traceback (most recent call last):
  File "/usr/src/paperless/src/src/django-q/django_q/cluster.py", line 454, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 154, in consume_file
    document = Consumer().try_consume_file(
  File "/usr/src/paperless/src/documents/consumer.py", line 264, in try_consume_file
    self.pre_check_file_exists()
  File "/usr/src/paperless/src/documents/consumer.py", line 106, in pre_check_file_exists
    self._fail(
  File "/usr/src/paperless/src/documents/consumer.py", line 90, in _fail
    raise ConsumerError(f"{self.filename}: {log_message or message}") from exception
documents.consumer.ConsumerError: Invoice-572B7446-0006.pdf: Cannot consume /tmp/paperless/paperless-upload-5jc7vf_c: File not found.
```

### Paperless-ngx version

1.9.0-dev

### Host OS

Debian GNU/Linux 10 (buster)

### Installation method

Docker - official image

### Browser

Brave, Chrome, Firefox

### Other

Paperless running in docker container behind nginx proxy

**Comment by stumpylog** [comment]
Do files in the consume folder work?

Basically, somehow the file either doesn't exist or has been removed.  Are you running with any configuration changes?

**Comment by shamoon** [comment]
I actually see this quite often on a test install I have. This install and my live install share the redis server and I have found that if I shut down the live install the error almost never happens. Not sure how to interpret that (i.e. if its a bug or unintended use, etc) and frankly Im curious if celery will fix it...

Any chance that applies to you @Lunaden ?

**Comment by Lunaden** [comment]
I don't have two install on same redis server. But Pb appears recently. So
maybe After some last updates or when i Moving from traefik to nginx proxy.
But don't see the link between this "bug" and the proxy i'm using.

**Comment by stumpylog** [comment]
My own instance also uses a reverse proxy and doesn't run into this.  Though I use Traefik.  So maybe there's some sort of buffering happening with nginx vs Traefik.

**Comment by shamoon** [comment]
I use Traefik too. I was trying to track it down but no real progress. With some extra log lines added it looks to me like `PostDocumentView` correctly creates a temp file, I see the files getting created in the container `/tmp/paperless` dir, and then pushes an `async_task` but I never actually see that task get inserted into the db, so `try_consume_file` is never called. Im just not clear where it bails out.

I do have a strong suspicion / hope this will be solved by celery, in the meantime we need a way to reproduce it (since OP is not using a shared redis, for example)

**Comment by Lunaden** [comment]
I took some time to debug more carefully.
During my migration from Traefik to nginx proxy I had renamed the paperless service of my stack.

```
version: "3.4"
services:
  paperless_web:


to 
version: "3.4"
services:
  paperless_web:
```

When I updated my stack afterwards I didn't pay attention but the service with the old name remained started... 
This way i having two containers with paperless running.

![image](https://user-images.githubusercontent.com/34348877/192965880-02e246fc-7cc2-4dbf-ac90-c428c63a0526.png)

I no longer have the problem after stopping and deleting the container of the old service...
